.github/ISSUE_TEMPLATE/4-REGULAR_RELEASE_CHECKLIST.md

```diff
@@ -1,7 +1,7 @@
 ---
 name: "\U0001F680 Regular Release Checklist"
 about: "\U0001F512 Maintainers only: create a checklist for a regular release process"
-title: ''
+title: 'Release checklist for release v2.x.x'
```

Comment:
I am not sure, if we should use `release` two times in the title. 

Can we use any one of the following? 
- `Release checklist for v2.x.x` 
- `Checklist for v2.x.x release`
- `Checklist for releasing v2.x.x`

Reply:
Addressed in https://github.com/wp-cli/wp-cli/pull/5168/commits/58e21ab0ee349afb06ddf97ecac9a6d9df57d198